cuqipy_fenics/geometry.py

```diff
@@ -177,12 +181,16 @@ class MaternExpansion(_WrappedGeometry):
 
     """
 
-    def __init__(self, geometry, length_scale, num_terms, boundary_conditions='Neumann', normalize=True): 
+    def __init__(self, geometry, length_scale, num_terms, nu=0.1, boundary_conditions='Neumann', normalize=True): 
+
+        if nu <= 0:
+            raise ValueError("Smoothness parameter nu must be positive")
+            
         super().__init__(geometry)
         if not hasattr(geometry, 'mesh'):
```

Comment:
Can you add a check here so that this geometry only works for FEniCS geoms. For now at least .)

Reply:
done 👍